### ISSUE TITLE
update GNOME runtime to 3.38

### ISSUE THREAD

**Issue by eszlari** [flathub/org.gnucash.GnuCash#35]

**Comment by fellen** [comment]
I just updated the [nightlies](https://code.gnucash.org/builds/flatpak/maint/?C=M;O=D) this week to 3.36. What are the benefits of 3.38?

**Comment by eszlari** [comment]
It's based on the new [freedesktop 20.08 runtime](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/release/20.08/NEWS). Also: new flatpak user have to download fewer runtimes.

**Comment by fellen** [comment]
My first trial to build with it aborted with:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LZ4_LIBS
    linked by target "libgroonga" in directory /run/build/mariadb/storage/mroonga/vendor/groonga/lib

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
Error: module mariadb: Der Kindprozess wurde mit Status 1 beendet

```

**Comment by eszlari** [comment]
The usual procedure is, to update the module and to search for patches used by distributions:
https://www.archlinux.org/packages/extra/x86_64/mariadb
https://packages.gentoo.org/packages/dev-db/mariadb
https://tracker.debian.org/pkg/mariadb-10.5

**Comment by Steinhagen** [comment]
This issue is tracked in: https://github.com/flathub/org.gnucash.GnuCash/pull/38

**Comment by pabl0** [comment]
BTW, recent upgrade to the 3.36 runtime gave the following warning:

Info: org.gnome.Platform is end-of-life, with reason: The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.

**Comment by fellen** [comment]
> BTW, recent upgrade to the 3.36 runtime gave the following warning:
> 
> Info: org.gnome.Platform is end-of-life, with reason: The GNOME 3.36 runtime is no longer supported as of February 13, 2021. Please ask your application developer to migrate to a supported platform.

In our nightlies we are already using 3.38, By an mistake in the documentation of our release process it was not merged into GnuCash 4.4.

But it will happpen in the next release comming weekend. Right, @jralls ?

**Comment by fellen** [comment]
Fixed with https://flathub.org/builds/#/builders/32/builds/43104
